R/subset_datarqst.R

```diff
@@ -37,66 +45,141 @@
 subset_datarqst <- function(file = file.choose(),
                             PopID = unique(pop_names$code),
                             Species = unique(Species_codes$Code),
-                            filter = NULL){
+                            filter = NULL,
+                            include_conflicting = FALSE,
+                            output_type = "R",
+                            return_R = FALSE,
```

Comment:
I agree, it's best to keep it as similar as possible. Checked back in run_pipelines, and it looks like there the R object is always returned. So, I have removed the "return_R" argument here, and made it so that subset_datarqst always returns the R object too.